packages/orbit-components/src/InputSelect/index.tsx

```diff
@@ -49,6 +49,7 @@ const InputSelect = React.forwardRef<HTMLInputElement, Props>(
       maxWidth,
       onKeyDown,
       spaceAfter,
+      readOnly = false,
```

Comment:
I've added an optioan prop `readOnly`, default value `false`.

Reply:
In case the viewport is small/medium mobile, it's not `readOnly` by default (this readOnly functionality was depended only on `isLargeMobile` value